app/templates/search/catalogue/index.html

```diff
@@ -34,7 +42,7 @@
           'small': True
         }) }}
       </div>
-      <ul class="tna-ul">
+      <ul class="tna-ul second-nav">
```

Comment:
There is a new secondary navigation component now.

It isn't yet documented in the Design System but you can see an example here: https://nationalarchives.github.io/tna-frontend/?path=/story/components-secondary-navigation--default

The Jinja template is done so you can import and use it like you have with the others if you like.

Let me know if you need help with this.

Reply:
You will need to update TNA Frontend (the npm package) as well as TNA Frontend Jinja to the latest versions.

In the dev container (in Docker Desktop) under the "Exec" tab, you can run:

```sh
. tna-vnm
npm install @nationalarchives/frontend@0.8.0
poetry add tna-frontend-jinja=0.8.0
```

Once they have updated, you can restart the application container by running this on your Mac:

```sh
docker compose up --build -d app
```